po/ro.po

```diff
@@ -0,0 +1,703 @@
+# Adrian Costin <adriancostin6@gmail.com>, 2021.
+msgid ""
+msgstr ""
+"Project-Id-Version: paru VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/Morganamilo/paru\n"
+"POT-Creation-Date: 2021-06-30 21:32+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Adrian Costin <adriancostin6@gmail.com>\n"
+"Language-Team: \n"
+"Language: ro\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: src/clean.rs:31
+msgid "Do you want to remove ALL AUR packages from cache?"
+msgstr "Doriți să stergeți din cache TOATE PACHETELE care provin din AUR?"
+
+#: src/clean.rs:33
+msgid "Do you want to remove all other AUR packages from cache?"
+msgstr "Doriți să stergeți din cache toate celelalte pachete AUR?"
+
+#: src/clean.rs:40
+msgid "Clone Directory: {}"
+msgstr "Clone Directory: {}"
+
+#: src/clean.rs:46
+msgid ""
+"\n"
+"Diff Directory: {}"
+msgstr ""
+"\n"
+"Diff Directory: {}"
+
+#: src/clean.rs:48
+msgid "Do you want to remove all saved diffs?"
+msgstr "Doriți să eliminați toate diff-urile salvate?"
+
+#: src/clean.rs:64
+msgid "can't open diff dir: {}"
+msgstr "nu se poate deschide directorul diff: {}"
+
+#: src/clean.rs:75 src/clean.rs:133
+msgid "could not remove '{}'"
+msgstr "nu se poate elimina '{}'"
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "nu se poate deschide directorul clone: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "opțiunea {} așteaptă o valoare"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "opțiunea {} nu permite alocarea unei valori"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "opțiune necunoscută --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "opțiune necunoscută -{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "nu s-a putut deschide fișierul cache '{}'"
+
+#: src/completion.rs:65
+msgid "could not update aur cache"
+msgstr "nu s-a putut actualiza cache-ul aur"
+
+#: src/config.rs:174
+msgid "invalid value '{val}' for key '{key}', expected: {exp}"
+msgstr ""
+"valoare incorectă '{val}' pentru cheia '{key}', valoare așteptată: {exp}"
+
+#: src/config.rs:474
+msgid "failed to find cache directory"
+msgstr "nu s-a găsit directorul cache"
+
+#: src/config.rs:477
+msgid "failed to find config directory"
+msgstr "nu s-a găsit directorul config"
+
+#: src/config.rs:501
+msgid "config file '{}' does not exist"
+msgstr "fișierul de configurare '{}' nu există"
+
+#: src/config.rs:550
+msgid "only one operation may be used at a time"
+msgstr "se poate folosi doar o singură operațiune la un anumit moment"
+
+#: src/config.rs:638
+msgid "can not find local repo '{}' in pacman.conf"
+msgstr "nu s-a putut găsi repo-ul local '{}' în pacman.conf"
+
+#: src/config.rs:670
+msgid "failed to initialize alpm: root={} dbpath={}"
+msgstr "nu s-a putut inițializa alpm: root={} dbpath{}"
```

Comment:
Missing =

Reply:
Saw the typo, fixed now, thanks for pointing it out.